<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerWidgetForm</class>
 <widget class="QWidget" name="LayerWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layers</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/3dglasses.png</normaloff>../images/3dglasses.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Project</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0" colspan="7">
       <widget class="QTableView" name="layerView"/>
      </item>
      <item row="0" column="0" colspan="7">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="lblProjectName">
          <property name="text">
           <string>&lt;unsaved&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../shift3.qrc">
            <normaloff>:/cut4/images/save.png</normaloff>:/cut4/images/save.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSaveAs">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../shift3.qrc">
            <normaloff>:/cut4/images/save_as.png</normaloff>:/cut4/images/save_as.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnOpen">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../shift3.qrc">
            <normaloff>:/cut4/images/open.png</normaloff>:/cut4/images/open.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="btnUp">
        <property name="text">
         <string>Up</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="btnDown">
        <property name="text">
         <string>Down</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="btnRemoveLayer">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnAddLayer">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="7">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Layer</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QPushButton" name="btnRightImage">
           <property name="text">
            <string>Right</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnLeftImage">
           <property name="text">
            <string>Left</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Scale</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <widget class="QLabel" name="lblRightImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>right Image</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QSpinBox" name="sbpShift">
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>x-Shift</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="spbScale">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QSpinBox" name="spbMoveX">
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>dx</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QSpinBox" name="spbMoveY">
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>dy</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="4">
          <widget class="QLabel" name="lblLeftImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>left Image</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Render</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QSpinBox" name="spbWidth">
        <property name="maximum">
         <number>32000</number>
        </property>
        <property name="value">
         <number>3840</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spbHeight">
        <property name="maximum">
         <number>32000</number>
        </property>
        <property name="value">
         <number>2160</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="cbxFormat">
        <item>
         <property name="text">
          <string>True Anaglyph</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gray Anaglyph</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Color Anaglyph</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HalfColor Anaglyph</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Optimized Anaglyph</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QPushButton" name="btnRender">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../shift3.qrc">
          <normaloff>:/cut4/images/camera.png</normaloff>:/cut4/images/camera.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spbWidth</tabstop>
  <tabstop>spbHeight</tabstop>
  <tabstop>btnRender</tabstop>
  <tabstop>cbxFormat</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnSaveAs</tabstop>
  <tabstop>btnOpen</tabstop>
  <tabstop>layerView</tabstop>
  <tabstop>btnAddLayer</tabstop>
  <tabstop>btnRemoveLayer</tabstop>
 </tabstops>
 <resources>
  <include location="../shift3.qrc"/>
 </resources>
 <connections/>
</ui>
